river/tree/_nodes/base.py

```diff
@@ -1,13 +1,14 @@
 import collections
+import copy
 import numbers
 import textwrap
+import typing
 from abc import ABCMeta, abstractmethod
-from typing import Dict, Iterator, List, Union
 
 from river import base
 from river.stats import Var
-from river.tree._attribute_test import AttributeSplitSuggestion  # noqa
-from river.tree._attribute_test import InstanceConditionalTest  # noqa
+
+from .._attribute_test import AttributeSplitSuggestion, InstanceConditionalTest
```

Comment:
I would call this simply `SplitSuggestion` instead of `AttributeSplitSuggestion`, but your call!

Reply:
I like that. I am so used to this code that I didn't realize it was a long name till you mentioned it. I'll rename it tomorrow :)

Reply:
Renamed in [2e63165](https://github.com/online-ml/river/pull/488/commits/2e63165b10a2c12af7ada79bc606cfb5c44805ad).